src/core/packages/di/server-internal/src/modules/index.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { ContainerModule } from 'inversify';
+import { loadCapabilites } from './capabilities';
+import { loadHttp } from './http';
+import { loadSavedObjects } from './saved_objects';
+
+export const core = new ContainerModule((options) => {
+  loadCapabilites(options);
+  loadHttp(options);
+  loadSavedObjects(options);
```

Comment:
nit: might be interesting to share the analytics client as well. But it can be done later.